extension/wrappers.h

```diff
@@ -329,4 +453,30 @@ CBasePlayer* UTIL_PlayerByIndex(int playerIndex)
 	return NULL;
 }
 
+inline CBaseEntity* GetContainingEntity(edict_t* pent)
```

Comment:
Can use `gameents->EdictToBaseEntity(edict_t*)`, no need for more boilerplate code.